cli/ts/commands/proveOnChain.ts

```diff
@@ -77,7 +77,11 @@ export const proveOnChain = async (
   const maciContractAddress = maciAddress || readContractAddress("MACI");
   const messageProcessorContractAddress = messageProcessorAddress || readContractAddress(`MessageProcessor-${pollId}`);
   const tallyContractAddress = tallyAddress || readContractAddress(`Tally-${pollId}`);
-  const subsidyContractAddress = subsidyAddress || readContractAddress(`Subsidy-${pollId}`);
+
+  let subsidyContractAddress;
```

Comment:
These changes seem unrelated from the rest of this PR - are these updates fixing some bug?

Reply:
yup, needed that to make manual test work properly so included here, but can definitely add in a separate one